docs/terms-and-conditions.md.js

```diff
@@ -59,7 +59,7 @@ In this document, when we refer to "**we**", "**us**" or "**our**", we mean Pari
 
 *General*
 
-  4.1  We provide software code in the App on an "open source" basis in accordance with the terms of the GPL 3.0 licence, which can be found here: [*https://opensource.org/licenses/GPL-3.0*](https://opensource.org/licenses/GPL-3.0), ("GPL 3.0") and make the source code of the App available here [*https://github.com/paritytech/parity-signer*](https://protect-eu.mimecast.com/s/G9QrCW6Agclr60pTxhQkw). You are free to use, modify and distribute the software code in the App in accordance with the terms of GPL 3.0. In the case of any conflict between the terms of GPL 3.0 and the terms of this EULA in relation to your use of the software code in the App, the terms of GPL 3.0 shall prevail.
```

Comment:
very surprised about that protect-eu.mimecast.com link, do you know anything about it?

Reply:
forgot to at you @kirushik

Reply:
Ouch!
It's some sort of redirect-tracker I am not aware about.
Thanks for spotting it, I'll investigate how did it end up here.

Reply:
It was introduced as a part of [this commit](https://github.com/paritytech/parity-signer/pull/504/commits/ef1f35218b15c11db654173b7ba9fd760e3ca12a), and committed by you from a Github Web UI (see the signature).

@Tbaut  Probably something one of your browser extensions doing? You definitely need to check your environment.

Reply:
I think it's most probably some kind of doc to markdown tool that I must have used to convert what I received from Fraser (a Google doc) because I didn't feel like formating it all per hand. This definitely defeated our review :(

Reply:
Removed the redirect link and pointed it to https://github.com/paritytech/parity-signer on Parity.io as well.